### PR TITLE
clean up ethers and web3js docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description/Original PRs

Original PR: https://github.com/moonbeam-foundation/moonbeam-docs/pull/734/

- use ethers.org instead of ethers.io
- format variable placeholder values
- use view more code snippet formatting
- update snippet paths

### CODE REVIEW

File: builders/build/eth-api/libraries/ethersjs.md [hunk 1]
```diff
@@ -7,48 +7,56 @@ description: 通过本教程以学习如何使用以太坊EtherJS代码库在Moo
 
 ## 概览 {: #introduction }
 
-[Ethers.js](https://docs.ethers.io/){target=_blank}代码库提供用户使用JavaScript与以太坊节点交互的多样工具，类似于Web3.js。Moonbeam拥有与以太坊相似的API供用户使用，其与以太坊风格的JSON RPC调用完全兼容。因此，开发者可以利用此兼容特性并使用Ethers.js库与Moonbeam节点交互，与在以太坊操作相同。您可以在此[博客文章](https://medium.com/l4-media/announcing-ethers-js-a-web3-alternative-6f134fdd06f3){target=_blank}了解更多关于Ethers.js的信息。
+[Ethers.js](https://docs.ethers.org/){target=_blank}代码库提供用户使用JavaScript与以太坊节点交互的多样工具，类似于Web3.js。Moonbeam拥有与以太坊相似的API供用户使用，其与以太坊风格的JSON RPC调用完全兼容。因此，开发者可以利用此兼容特性并使用Ethers.js库与Moonbeam节点交互，与在以太坊操作相同。您可以在其[官方文档](https://docs.ethers.org/v6/){target=_blank}获取更多关于如何使用Ethers.js的信息。
+
+您可以在此[博客文章](https://medium.com/l4-media/announcing-ethers-js-a-web3-alternative-6f134fdd06f3){target=_blank}了解更多关于Ethers.js的信息。
```
Comment (albertov19) [inline builders/build/eth-api/libraries/ethersjs.md:12]: Didn't I changed this to remove this paragraph. I think this is a bit unnecessary now hehe

**Review by albertov19 — APPROVED** [review]
LGTM! @themacexpert can you check this renders correctly please

File: builders/build/eth-api/libraries/ethersjs.md (hunk 1 above)
Comment (eshaben) [reply builders/build/eth-api/libraries/ethersjs.md:12]: Oh yeah I split it out with the intention of deleting it and never did. Good catch!

File: builders/build/eth-api/libraries/ethersjs.md [hunk 2]
```diff
@@ -7,48 +7,54 @@ description: 通过本教程以学习如何使用以太坊EtherJS代码库在Moo
...
  --8<-- 'text/faucet/faucet-list-item.md'
- - 
+ -
 --8<-- 'text/common/endpoint-examples.md'
 
 !!! 注意事项
     --8<-- 'text/common/assumes-mac-or-ubuntu-env.md'
 
-## 创建一个JavaScript项目 {: #create-a-javascript-project }
+安装Scaffold-ETH
```
Comment (themacexpert) [inline builders/build/eth-api/libraries/ethersjs.md:26]: Just double checking that this scaffold eth mention is correct? I don't see it on the English page.

**Review by themacexpert — APPROVED** [review]
Couple things to fix other than that great

File: builders/build/eth-api/libraries/ethersjs.md (hunk 2 above)
Comment (eshaben) [reply builders/build/eth-api/libraries/ethersjs.md:26]: Haha nope, I added it there to copy the first two characters. Good catch!

**Pull request merged**